Allow levelbuilders to get selected block XML

mikeharv opened the pull request:
Add a button to levelbuilder's Extra Links for copying a selected Blockly Block.

![image](https://user-images.githubusercontent.com/43474485/150384505-35ec0929-5182-4722-bf68-732634f582fd.png)

This button gets the full XML for the currently selected Blockly block, if there is one. This feature will be invaluable for K5 curriculum writers who need to add specific blocks to level instructions or hints. 

Example usage:
Pressing the button with the background block here selected:
![image](https://user-images.githubusercontent.com/43474485/150381691-167db8aa-5229-40bb-8610-149cdf502244.png)

Generates and copies this XML:
```<block type="gamelab_setBackgroundImageAs"> <field name="IMG">"hw_plate_background_1"</field> <next> <block type="gamelab_playSound"> <field name="SOUND">Choose</field> </block> </next> </block>```

dmcavoy reviewed (approved):
I don't know much about the blockly blocks part but this seems good to me. One nit: it would be great if the buttons were not on the same line.

mikeharv commented:
> I don't know much about the blockly blocks part but this seems good to me. One nit: it would be great if the buttons were not on the same line.

Great idea! Added a line break and updated screenshot

davidsbailey reviewed (commented):
Wow, cool! This looks like a good approach to me, but I think I would like someone on @code-dot-org/starlabs to take a peek too.

ajpal commented on apps/src/sites/studio/pages/levelbuilder.js:
```diff
@@ -49,6 +49,15 @@ window.levelbuilder.copyWorkspaceToClipboard = function() {
 };
 
+window.levelbuilder.copySelectedBlockToClipboard = function() {
+  if (Blockly.blockly_.selected) {
```
Can you add `selected` as a read-only property here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/cdoBlocklyWrapper.js (similar to https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/googleBlocklyWrapper.js#L142), then change this line and below to just `Blockly.selected`?
